src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss} %-5p: %m%n"/>
+        </Console>
+    </Appenders>
+    <Loggers>
+        <Root level="DEBUG">
+            <AppenderRef ref="Console"/>
+        </Root>
+    </Loggers>
+</Configuration>
```

Comment:
What does this do?

Reply:
no idea

Reply:
It configures log4j2 logging system to print its logs to the console.

We use slf4j loggers in some classes, so this forwards the logs conveniently to the console. Except it requires log4j2-slf4j bridge to work. That is one line in build.gradle that I just removed from my Scala 3 PR because it was unrelated to the PR and it caused Snyk to warn about vulnerabilities of log4j2's dependencies (some version of junit).

Reply:
Alright thanks! Does it also apply to error logs? And does it replace the current mechanism where error logs are shown inside IDEA with the blinking exclamation mark, or is it just in addition to that?